device/common/profiles/th2/7260/BALANCED/pg_profile_lookup.ini

```diff
@@ -0,0 +1,8 @@
+# PG lossless profiles.
```

Comment:
please include the 50g speed and 5m,40m,300m lines as well

Reply:
Updated with copying 100G data (the original 50G data was higher than 100G config.